qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/2-client.yaml

```diff
@@ -1,3 +0,0 @@
-tasks:
-- kclient:
-- print: "**** done client"
```

Comment:
So I'm looking at this change again. Does this PR produce any functional difference? Is the goal to start testing all kernel branches (stock/testing) we normally do?

Please note that premerge/postmerge scripts are not really free. Structuring the matrix this way can blow up the matrix with jobs that will always be pruned.

Reply:
I think we don't need this PR any more. The code changed a lot, but not sure why it didn't generated any conflicts in this PR.

The new code has added the `ceph-fuse` client.